Doc/library/sqlite3.rst

```diff
@@ -198,7 +198,8 @@ Module functions and constants
 
    *detect_types* defaults to 0 (i. e. off, no type detection), you can set it to
    any combination of :const:`PARSE_DECLTYPES` and :const:`PARSE_COLNAMES` to turn
-   type detection on.
+   type detection on. Due to SQLite behaviour, types can't be detected for generated
+   fields (for example ``max(data)``), even when *detect_types* parameter is set.
```

Comment:
I think you should add what happens in this case. From the original bpo, it sounds like it returns a `str` (I have not verfied this).

Reply:
I checked with the reproducer.py file attached to the bug report and I confirm it returns a `str`. I added a new commit for that.